osu.Game/OsuGame.cs

```diff
@@ -671,6 +671,8 @@ protected override void LoadComplete()
                 };
             }
 
+            overlays.AddRange(new OverlayContainer[] { login, nowPlaying });
```

Comment:
this may be changing the behaviour of `CloseAllOverlays` (see the `bool` in the method). needs further investigation.

Reply:
We can basically revert https://github.com/ppy/osu/pull/4765 as closing the now playing overlay should be done even when the bool is false.